lib/optimizely/helpers/validator.rb

```diff
@@ -153,9 +153,11 @@ def boolean?(value)
       def same_types?(value_1, value_2)
         # Returns true if given values are of same types.
         #         false otherwise.
+        # Numeric values are considered as same type.
+
+        return true if value_1.is_a?(Numeric) && value_2.is_a?(Numeric)
```

Comment:
Looks like it's okay to do the numeric check before the boolean check, in Ruby. I've confirmed that `true` and `false` are not `Numeric`s.